--- origsrc/cgit-0.9.0.3/Makefile	2012-03-18 05:16:27.000000000 -0500
+++ src/cgit-0.9.0.3/Makefile	2012-11-07 15:00:33.551178600 -0600
@@ -1,7 +1,7 @@
 CGIT_VERSION = v0.9.0.3
 CGIT_SCRIPT_NAME = cgit.cgi
-CGIT_SCRIPT_PATH = /var/www/htdocs/cgit
-CGIT_DATA_PATH = $(CGIT_SCRIPT_PATH)
+CGIT_SCRIPT_PATH = /srv/www/cgi-bin
+CGIT_DATA_PATH = /srv/www/htdocs
 CGIT_CONFIG = /etc/cgitrc
 CACHE_ROOT = /var/cache/cgit
 prefix = /usr
@@ -45,7 +45,6 @@ uname_O := $(shell sh -c 'uname -o 2>/de
 uname_R := $(shell sh -c 'uname -r 2>/dev/null || echo not')
 
 ifeq ($(uname_O),Cygwin)
-	NO_STRCASESTR = YesPlease
 	NEEDS_LIBICONV = YesPlease
 endif
 
--- origsrc/cgit-0.9.0.3/git/Makefile	2011-01-30 21:02:37.000000000 -0600
+++ src/cgit-0.9.0.3/git/Makefile	2012-11-07 14:59:07.011059200 -0600
@@ -916,12 +916,10 @@ ifeq ($(uname_O),Cygwin)
 	endif
 	NEEDS_LIBICONV = YesPlease
 	NO_FAST_WORKING_DIRECTORY = UnfortunatelyYes
-	NO_TRUSTABLE_FILEMODE = UnfortunatelyYes
 	NO_ST_BLOCKS_IN_STRUCT_STAT = YesPlease
 	# There are conflicting reports about this.
 	# On some boxes NO_MMAP is needed, and not so elsewhere.
 	# Try commenting this out if you suspect MMAP is more efficient
-	NO_MMAP = YesPlease
 	X = .exe
 	COMPAT_OBJS += compat/cygwin.o
 	UNRELIABLE_FSTAT = UnfortunatelyYes
--- origsrc/cgit-0.9.0.3/git/compat/cygwin.c	2011-01-30 21:02:37.000000000 -0600
+++ src/cgit-0.9.0.3/git/compat/cygwin.c	2012-11-07 14:57:33.676893400 -0600
@@ -1,6 +1,8 @@
 #define WIN32_LEAN_AND_MEAN
-#include "../git-compat-util.h"
+#include <errno.h>
+#include <fcntl.h>
 #include "win32.h"
+#include "../git-compat-util.h"
 #include "../cache.h" /* to read configuration */
 
 static inline void filetime_to_timespec(const FILETIME *ft, struct timespec *ts)
