--- origsrc/cgit-0.9.1/filters/syntax-highlighting.sh	2012-11-14 18:28:59.000000000 -0600
+++ src/cgit-0.9.1/filters/syntax-highlighting.sh	2012-11-14 19:27:03.475381200 -0600
@@ -53,7 +53,7 @@ EXTENSION="${BASENAME##*.}"
 # found (for example) on EPEL 6.
 #
 # This is for version 2
-exec highlight --force -f -I -X -S "$EXTENSION" 2>/dev/null
+#exec highlight --force -f -I -X -S "$EXTENSION" 2>/dev/null
 
 # This is for version 3
-#exec highlight --force -f -I -O xhtml -S "$EXTENSION" 2>/dev/null
+exec highlight --force -f --inline-css -O xhtml -S "$EXTENSION" 2>/dev/null
